docs/changes/5.3.0.md

```diff
@@ -13,6 +13,7 @@ Release date: `2021-xx-xx`
 ### Direct Transfer
 
 - [NXDRIVE-2](https://jira.nuxeo.com/browse/NXDRIVE-2):
+- [NXDRIVE-2737](https://jira.nuxeo.com/browse/NXDRIVE-2737): Fix Minimize/maximize issue on Direct Transfer screen
```

Comment:
Move to GUI section as it is more an interface update than a feature modification.